src/pages/Job/JobDetail.tsx

```
@@ -0,0 +1,330 @@
+  companyRevenue: "1900000000",
+  companyAddress:
+    "서울특별시 강남구 논현로87길 15 서울특별시 강남구 역삼동 736-54 LS빌딩",
+  companyWebsite: "www.icnkorea.com",
+  languageRequirement: "필수 영어(중상) : 영어로 업무 가능",
+  restDay: "협의",
+  visaType: "인턴비자",
+  decisionDate: "2025-05-20",
+};
+
+const JobDetail = () => {
+  const { id } = useParams<{ id: string }>();
```

ESLint

[error] 'id' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)